python/hsfs/training_dataset.py

```diff
@@ -633,7 +633,7 @@ def get_query(self, online: bool = True, with_label: bool = False):
             `str`. Query string for the chosen storage used to generate this training
                 dataset.
         """
-        return self._training_dataset_engine.query(self, online, with_label)
+        return self._training_dataset_engine.query(self, online, with_label, engine.get_type())
```

Comment:
Doesn't the `get_type()` return a string with `"hive"` and not a boolean?

Reply:
@moritzmeister Absolutely! just fixed that.